foreign-key.md

```diff
@@ -318,3 +318,28 @@ Create Table | CREATE TABLE `child` (
 ### Compatibility with MySQL
 
 When you create a foreign key without specifying a name, the name generated by TiDB is different from that generated by MySQL. For example, the foreign key name generated by TiDB is `fk_1`, `fk_2`, and `fk_3`, while the foreign key name generated by MySQL is `table_name_ibfk_1`, `table_name_ibfk_2`, and `table_name_ibfk_3`.
+
+Both MySQL and TiDB parse but ignore "inline `REFERENCES` specifications". Only `REFERENCES` specifications that are part of `FOREIGN KEY` definitions are checked and enforced. For example:
+
+```sql
+CREATE TABLE parent (
+    id INT KEY
+);
+
+CREATE TABLE child (
+    id INT,
+    pid INT REFERENCES parent(id)
+);
+
+SHOW CREATE TABLE child;
++-------+-------------------------------------------------------------+
+| Table | Create Table                                                |
++-------+-------------------------------------------------------------+
+| child | CREATE TABLE `child` (                                      |
+|       |   `id` int(11) DEFAULT NULL,                                |
+|       |   `pid` int(11) DEFAULT NULL                                |
+|       | ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
++-------+-------------------------------------------------------------+
+```
+
+The above output of `SHOW CREATE TABLE` indicates that no foreign keys exist in the `child` table.
```

Comment:
````suggestion
Both MySQL and TiDB parse but ignore "inline `REFERENCES` specifications". Only `REFERENCES` specifications that are part of `FOREIGN KEY` definitions are checked and enforced. The following example creates a foreign key constraint using a `REFERENCES` clause:

```sql
CREATE TABLE parent (
    id INT KEY
);

CREATE TABLE child (
    id INT,
    pid INT REFERENCES parent(id)
);

SHOW CREATE TABLE child;
```

The output shows that the `child` table does not contain any foreign keys:

```sql
+-------+-------------------------------------------------------------+
| Table | Create Table                                                |
+-------+-------------------------------------------------------------+
| child | CREATE TABLE `child` (                                      |
|       |   `id` int(11) DEFAULT NULL,                                |
|       |   `pid` int(11) DEFAULT NULL                                |
|       | ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
+-------+-------------------------------------------------------------+
```
````

Reply:
@dveeden Don't use GitHub suggestion to commit here, since the last sentence was missed again.

> The above output of `SHOW CREATE TABLE` indicates that no foreign keys exist in the `child` table.

Reply:
@crazycs520 I think @Oreoxmt moved that to be above the output isn't it?